src/components/SearchBar.tsx

```diff
@@ -1,21 +1,37 @@
-import { TextField as MaterialTextField } from '@material-ui/core';
+import { TextField as MaterialTextField, makeStyles, Theme } from '@material-ui/core';
+import { createStyles } from '@material-ui/core/styles';
 import React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
 
 interface SearchProps {
   onSearchChange: (event: React.ChangeEvent<{ value: unknown }>) => void;
+  onSearchExit?: any;
   placeholder?: string;
 }
 
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
Also since this isn't actually depending on the props in any way, I'd convert this to a `withStyles(styles)(SearchBar)` format that we use in most other places that don't depend on props.